src/main/java/net/comments/objects/BCommentTable.java

```diff
@@ -0,0 +1,43 @@
+package net.comments.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BCommentTable implements CommentTable {
+    private final WebDriver driver;
+    private List<Comment> commentsList;
+
+    public BCommentTable(WebDriver driver) {
+        this.driver = driver;
+        commentsList = new ArrayList<Comment>();
+    }
+
+    public void selectComment(int commentsNumber) {
+        this.driver.findElements(By.name("SelectedId")).get(commentsNumber).click();
+    }
+
+    public void fillTable() {
```

Comment:
In order to read comments from HTML page, I need to use a sequence of methods in the specific order.
Why `getTableComments`isn't able to read comments and provide them?

Reply:
"In order to read comments from HTML page, I need to use a sequence of methods in the specific order." - Sorry, but I cannot understand this comment. What do you mean? 

Maybe I need to re-write By.name("SelectedId")).get(commentsNumber) to By.xpath(//*[@name = "SelectedId" and @value = 1])

Reply:
"Why getTableCommentsisn't able to read comments and provide them?" - Should I create separate method which can read one comments and return it?

Reply:
> "In order to read comments from HTML page, I need to use a sequence of methods in the specific order." - Sorry, but I cannot understand this comment. What do you mean?

```java
table. fillTable()
List<Comment> comments = table.getTableComments()
```

Why I can't just use `List<Comment> comments = table.getTableComments()`?